python/pyspark/sql/functions.py

```diff
@@ -10403,6 +10405,84 @@ def udf(
         return _create_py_udf(f=f, returnType=returnType, useArrow=useArrow)
 
 
+def udtf(
```

Comment:
Oh, BTW all Python API have to be listed in `python/docs/source/reference/pyspark.sql/*.rst` files.